mmdet/models/necks/yolo_neck.py

```diff
@@ -11,31 +11,36 @@
 class DetectionBlock(nn.Module):
     """Detection block in YOLO neck.
 
-    Let out_channels = n, the DetectionBlock contains:
-    Six ConvLayers, 1 Conv2D Layer and 1 YoloLayer.
-    The first 6 ConvLayers are formed the following way:
-        1x1xn, 3x3x2n, 1x1xn, 3x3x2n, 1x1xn, 3x3x2n.
-    The Conv2D layer is 1x1x255.
-    Some block will have branch after the fifth ConvLayer.
-    The input channel is arbitrary (in_channels)
-
-    Args:
-        in_channels (int): The number of input channels.
-        out_channels (int): The number of output channels.
-        conv_cfg (dict): Config dict for convolution layer. Default: None.
-        norm_cfg (dict): Dictionary to construct and config norm layer.
-            Default: dict(type='BN', requires_grad=True)
-        act_cfg (dict): Config dict for activation layer.
-            Default: dict(type='LeakyReLU', negative_slope=0.1).
+        Let out_channels = n, the DetectionBlock normally contains 5 ConvModules,
+        Their sizes are 1x1xn, 3x3x2n, 1x1xn, 3x3x2n, and 1x1xn respectively.
+        If the spp is on, the DetectionBlock contains 6 ConvModules and
+        3 pooling layers, sizes are 1x1xn, 3x3x2n, 1x1xn,
+        5x5 maxpool, 9x9 maxpool, 13x13 maxpool, 1x1xn, 3x3x2n, 1x1xn.
+        The input channel is arbitrary (in_channels)
+    yolo
+        Args:
+            in_channels (int): The number of input channels.
+            out_channels (int): The number of output channels.
+            conv_cfg (dict): Config dict for convolution layer. Default: None.
+            spp_on (bool): whether to integrate a spp module. Default: False.
+            spp_pooler_sizes (tuple): A set of sizes for spatial pyramid pooling.
```

Comment:
Could we use SPP when `spp_pooler_sizes` is not None?

Reply:
We may rename `spp_pooler_sizes` to `spp_scales`.

Reply:
> Could we use SPP when spp_pooler_sizes is not None?
Yes, the code should be able to work with custom spp_scales.